src/app/containers/CpsText/index.jsx

```diff
@@ -1,43 +1,34 @@
 import React from 'react';
-import { arrayOf, oneOfType, shape, oneOf, string } from 'prop-types';
+import { arrayOf, shape, string } from 'prop-types';
 import paragraph from '../Paragraph';
-import unorderedList, { ListPropTypes } from '../BulletedList';
-import { ParagraphPropTypes } from '../BulletedListItem';
+import unorderedList from '../BulletedList';
 import Blocks from '../Blocks';
 
-// Render orderedLists as unorderedLists for now.
-const componentsToRender = {
-  paragraph,
-  unorderedList,
-  orderedList: unorderedList,
-};
-
-const CpsTextContainer = ({ blocks }) => {
+const CpsTextContainer = ({ blocks, componentsToRender }) => {
   if (!blocks) return null;
 
   return <Blocks blocks={blocks} componentsToRender={componentsToRender} />;
 };
 
 export const CpsTextPropTypes = {
   blocks: arrayOf(
-    oneOfType([
-      shape({
-        type: oneOf(['unorderedList', 'orderedList']).isRequired,
-        model: shape(ListPropTypes),
-      }),
-      shape({
-        type: oneOf(['paragraph']),
-        model: shape({
-          text: string,
-          ...ParagraphPropTypes,
-        }),
-      }),
-    ]).isRequired,
+    shape({
+      type: string.isRequired,
+    }),
   ).isRequired,
 };
 
 CpsTextContainer.propTypes = {
   ...CpsTextPropTypes,
+  componentsToRender: shape(),
```

Comment:
Did some googling for this - when we want to check the type of the value, but we don't know the keys, the `objectOf` function seems to be the way

https://reactjs.org/docs/typechecking-with-proptypes.html

```suggestion
  componentsToRender: objectOf(elementType),
```